nextflow_schema.json

```diff
@@ -5,36 +5,42 @@
     "description": "Differential gene expression analysis and pathway analysis of RNAseq data",
     "type": "object",
     "definitions": {
-        "mandatory_arguments": {
-            "title": "Mandatory arguments",
+        "main_arguments": {
+            "title": "Main input arguments",
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "properties": {
+                "input_type": {
+                    "type": "string",
+                    "default": "featurecounts",
+                    "description": "Which type of input is provided, one of featurecounts, rsem, salmon"
+                },
                 "gene_counts": {
                     "type": "string",
                     "description": "If input_type = featurecounts: Raw count table (TSV). Columns are samples and rows are genes. 1st column Ensembl_ID, 2nd column gene_name. If input_type = rsem or input_type = salmon: Path to rsem/salmon output folder."
                 },
-                "metadata": {
+                "input": {
                     "type": "string",
-                    "description": "Metadata table (TSV). Rows are samples and columns contain sample grouping."
+                    "description": "Metadata table/samplesheet (TSV). Rows are samples and columns contain sample grouping."
                 },
                 "model": {
                     "type": "string",
-                    "description": "Linear model function to calculate the contrasts (TXT). Variable names should be columns in metadata file."
+                    "description": "Linear model function to calculate the contrasts (TXT). Variable names should be columns in metadata file/samplesheet."
                 },
-                "project_summary": {
+                "gtf": {
                     "type": "string",
-                    "description": "Project summary file downloaded from the qPortal."
+                    "description": "Not necessary if genome is set or input_type is featurecounts. GTF file to be used for DESeq if input is rsem or salmon."
                 },
-                "multiqc": {
+                "software_versions": {
                     "type": "string",
-                    "description": "multiqc.zip folder containing the multiQC plots and report."
+                    "default": "None",
+                    "description": "CSV containing nf-core/rnaseq software versions."
                 }
             },
-            "required": ["metadata", "model", "project_summary", "multiqc", "gene_counts"]
+            "required": ["input_type", "gene_counts", "input", "model", "versions"]
```

Comment:
```suggestion
            "required": ["input_type", "gene_counts", "input", "model", "software_versions"]
```

Reply:
have you run `nf-core schema-build` ? That should have been updated then

Reply:
just did, but it did not update the param for some reason. I updated it manually